eng/common/pipelines/templates/steps/create-pull-request.yml

```diff
@@ -16,8 +16,8 @@ parameters:
 steps:
 
 - pwsh: |
-    echo "git add ."
-    git add .
+    echo "git add -A ."
+    git add -A .
```

Comment:
This is a good fix as we were missing deletions. I think this can be change to simply `git add -A` without the `.`. https://stackoverflow.com/questions/572549/difference-between-git-add-a-and-git-add/572660#572660 has a great demonstration about how this works.